internal/hcs/system.go

```diff
@@ -1,6 +1,7 @@
 package hcs
 
 import (
+	gcontext "context"
```

Comment:
Just use `context`?

Reply:
Cant it collies with some of the hcs* parameters